warehouse/query-core/src/test/java/datawave/query/CompositeFunctionsTest.java

```diff
@@ -182,6 +183,8 @@ public void setup() {
         tldEventQueryLogic.setFullTableScanEnabled(true);
         tldEventQueryLogic.setMaxDepthThreshold(6);
         deserializer = new KryoDocumentDeserializer();
+        this.eventQueryLogic.getConfig().setLocatorSupplier(InMemoryTabletLocator::new);
```

Comment:
Would the setter ever be used in prod?  Spring provides ReflectionTestUtils for injecting classes. Would it be more palatable to use that to inject an in memory tablet locator or would that be too ugly/failure prone?